prometheus/operator/base/jupyterhub-servicemonitor.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  labels:
+    app: jupyterhub
```

Comment:
Would changing the name to odh-jupyterhub be safer?

Reply:
I think jupyterhub is better since all other JH components have this label

Reply:
Fine by me. I'll defer to @nakfour

Reply:
The label is bound to the `team` right? In that case, I think we're fine

Reply:
@anishasthana is this supposed to scrape metrics from KF jupyter ? If not, we need to say it is odh-jupyterhub. I know KF does not have jupyterhub but there is always confusion between what odh has and what KF has.

Reply:
I think a lot of ODH [JH components](https://github.com/opendatahub-io/odh-manifests/blob/master/jupyterhub/jupyterhub/base/jupyterhub-dc.yaml#L4-L5) have this label. 
Are we planning to rename them to odh-jupyterhub?

Reply:
Not particular plan, we have other labels too, if you want to be more specific `component.opendatahub.io/name: jupyterhub`

Reply:
I think we should keep it as app=jypyterhub rn and create a new PR to update labels for all the JH components.

Reply:
ok thats fine, please open the issue to change all labels to jupyterhub